src/agent.rs

```diff
@@ -314,7 +395,7 @@ struct JobBridge<AGN: Agent> {
 
 impl<AGN: Agent> Bridge<AGN> for JobBridge<AGN> {
     fn send(&mut self, msg: AGN::Input) {
-        let upd = AgentUpdate::Input(msg, SINGLETON_ID);
+        let upd = AgentUpdate::Input(msg, None);
```

Comment:
Not sure about removing the SINGLETON_ID stuff here, as it is still used elsewhere.